src/app/models/AudioRecording.ts

```diff
@@ -12,17 +18,26 @@ import { AbstractModel } from "./AbstractModel";
 export interface AudioRecordingInterface {
   id?: Id;
   uuid?: Uuid;
+  uploaderId?: Id;
   recordedDate?: DateTimeTimezone | string;
   siteId?: Id;
-  durationSeconds?: number;
+  durationSeconds?: Duration | number;
   sampleRateHertz?: number;
   channels?: number;
   bitRateBps?: number;
   mediaType?: string;
   dataLengthBytes?: number;
+  fileHash?: string;
   status?: "ready" | "uploading" | "corrupt";
+  notes?: Notes | string;
```

Comment:
should only every be a string in a bug case - remove from type and add conversion in constructor